include/exadg/aero_acoustic/single_field_solvers/acoustics.h

```diff
@@ -80,6 +117,26 @@ class SolverAcoustic
 
   // postprocessor
   std::shared_ptr<Acoustics::PostProcessorBase<dim, Number>> postprocessor;
+
+private:
+  double
+  compute_sub_time_step_size(double const global_dt)
+  {
+    double const current_dt = time_integrator->get_time_step_size();
```

Comment:
Does `time integrator->get_time_step_size()` return a sub-time step here, not a global time step? If so, I think the name `current_dt` should rather be `sub_dt`. I also think implementing the content of this function at the place where this function is called would actually make the code easier to understand. The reason is that I originally thought the function name `compute_sub_time_step_size()` implies this function computes a sub-time step given a global time step, like `sub_dt = global_dt / n`, which does not seem to be the case here. I rather understand this function as `get_and_potentially_adjust_sub_time_step()`. It seems as if this function is only used once, hence, the suggestion to maybe move the code there.

Reply:
[0c319a8](https://github.com/exadg/exadg/pull/645/commits/0c319a8489f99e5a4850613c92030e09e39c27e1)